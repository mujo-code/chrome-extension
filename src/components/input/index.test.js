import React from 'react'
import Renderer from 'react-test-renderer'
import { Input } from '.'

test('Input should match snapshot', () => {
  const tree = Renderer.create(<Input />).toJSON()
  expect(tree).toMatchSnapshot()
})
