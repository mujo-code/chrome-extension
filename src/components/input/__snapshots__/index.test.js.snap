// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should match snapshot 1`] = `
<div
  className="css-j7qwjs"
>
  <label
    className="css-1w1bss4"
    color="mischka"
    htmlFor="label-is-required"
  >
    <small
      className="css-0"
    >
      Label is Required
    </small>
  </label>
  <input
    backgroundColor="mischka"
    className="css-yd3hjj"
    color="outerSpace"
    id="label-is-required"
    type="text"
  />
</div>
`;
